smart-campus-service-impl/src/main/java/hu/unideb/smartcampus/service/ejabberd/EjabberdService.java

```diff
@@ -0,0 +1,20 @@
+package hu.unideb.smartcampus.service.ejabberd;
+
+/**
+ * Class for every service which indicates the ckass will control the Ejabberd XMPP server via REST
+ * API.
+ *
+ */
+public class EjabberdService {
+
+  /**
+   * Ejabberd REST API host.
+   */
+  protected static final String HOST = "http://smartcampus:5280/api";
```

Comment:
In my view, it can be more configurable if you define these properties in a property file.
After that, you can create a Configuration class which will read these properties.

(Link: http://www.baeldung.com/properties-with-spring)

After, you can create a map and upload it with these properties.
The map should be immutable + (static or injectable).